pallets/identity/src/lib.rs

```diff
@@ -520,13 +520,14 @@ decl_module! {
 
             match &claim {
                 Claim::CustomerDueDiligence(..) => Self::base_add_cdd_claim(target, claim, issuer, expiry)?,
-                Claim::InvestorZKProof(_t, _s, cdd_id, _p) => {
+                Claim::InvestorZKProof(_s, _s_id, cdd_id, _p) => {
```

Comment:
Please define a new extrinsic for adding `InvestorZKProof` claims. These claims are a lot more expensive to add than others and I don't want set the weight of `add_claims` extrinsic to cover that.

ps Refunds are (almost) never a good option.

Reply:
Yep, We talked about this in yesterday standup to have another extrinsic for that & decided to cover this in another task.